### DISCUSSION TITLE
Youtube

### DISCUSSION THREAD

**Discussion by wlshedd** [baffler/Transparent-Twitch-Chat-Overlay#184] [Q&A]
so i know youtube works with this but im so dumb and cant figure it out, i read what it told me to do but i barely know what any of the words mean so if someone could help that would be great thanks!

**Comment by bloodmoon14** [comment]
Here something that will help you
https://github.com/baffler/Transparent-Twitch-Chat-Overlay/wiki#how-do-i-get-youtube-gaming-chatany-other-streaming-service

**Reply by finalmakerr** [reply]
**That's ChatGPT answer to the console log for YouTube hopefully there is a fix for YouTube live chat not working, it's probably related to YouTube API or streamelements!?**


CORS Policy Issue:

csharp
Copy code
Access to XMLHttpRequest at 'https://api.streamelements.com/kappa/v3/emotes/648a32b1a4e051fb6107b93f' from origin 'https://streamelements.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
This error suggests that there is a Cross-Origin Resource Sharing (CORS) policy issue when trying to fetch emotes from StreamElements. This could be preventing some parts of the chat functionality from working correctly.

Strict MIME Type Checking:

python
Copy code
Refused to apply style from 'https://unpkg.com/animal-crossing@8.2.0/lib/index.js' because its MIME type ('application/javascript') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
This message indicates that a JavaScript file is being blocked because it's not being served with the correct MIME type. If this file is required for rendering the chat interface or handling chat events, its absence could cause the chat not to work.
